### PR TITLE
Sound

### LINKED ISSUE #2: remote crickets activation

### LINKED ISSUE #3: remote airhorn activation

### PR DESCRIPTION
**Pull request by bifurcation**
Fixes #2 
Fixes #3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -93,6 +97,30 @@
   "reload": function(obj) {
     window.location.reload();
   },
+
+  // Play an airhorn sound
+  "airhorn": function(obj) {
+    var command = obj.message.replace(/^\/airhorn[ ]+/, "");
+    console.log(command)
```
Comment (martinthomson) [inline index.html]: semi

File: index.html [hunk 2]
```diff
@@ -93,6 +97,30 @@
...
+    switch (command) {
+      case "start": airhorn.start(true, 0.24, 0.34); break;
+      case "stop": airhorn.stop(); break;
+      default: airhorn.start(false); break;
+    }
+    return defaultDisplay(obj);
+  },
+
+  // Play a cricket sound
+  "crickets": function(obj) {
```
Comment (martinthomson) [inline index.html]: no quotes on dictionary member names needed

File: sound/sound.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+// Adapted from https://airhorner.com/scripts/main.min.js
```
Comment (martinthomson) [inline sound/sound.js]: Inline this please.

**Comment by martinthomson** [comment]
r+ with comments addressed

File: index.html (hunk 2 above)
Comment (bifurcation) [inline index.html]: I'm going to keep them -- you need them if it's a regex, so best be consistent.

**Pull request merged**